CornAPI/Wallet/HLAPI/ImplementedWalletMethods.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+
+namespace CornAPI.Wallet.HLAPI
+{
+    /// <summary>
+    /// Lists all implemented wallet methods on the high level api as constants for Attribute access
+    /// </summary>
+    public enum ImplementedWalletMethods
```

Comment:
maybe we should go ahead and add all methods to this just so we have them for the future, maybe not